src/fxos-radio.js

```diff
@@ -37,28 +35,23 @@ module.exports = component.register('gaia-radio', {
 
     // Bind label listeners in the next turn
     // to make sure that HTML has been parsed.
-    setTimeout(() => {
-      this.bindLabels();
-      this.makeAccessible();
```

Comment:
@yzen I found that having this async was causing Marionette a11y failures/intermittents. I'm not sure why it was inside `setTimeout`, but if we want to move it back there we'll need to make sure tests aren't run before it's complete.